src/screens/SendTokens.js

```diff
@@ -234,6 +237,8 @@ class SendTokens extends React.Component {
         this.data = data;
         $('#pinModal').modal('show');
       } else {
+        // currently we only support HTR
+        data.tokens = [];
```

Comment:
I feel we should make this verification beforehand and show an error message accordingly. Here we could assert that `data.tokens == []`.

Reply:
I don't know how this would affect the software wallet, but we could add it to `getData` method on `src/components/SendTokensOne.js`.

Reply:
You misunderstood me. I don't think we should do `data.tokens = []` at all. We should check whether the list of tokens is empty and raise an exception if it is not.